### PR TITLE
merge common-public back into common

### PR DESCRIPTION
**Pull request by wentokay**
removes common-public by merging all of its code into common

### CODE REVIEW

File: .github/workflows/pull_requests_and_merges.yml [hunk 1]
```diff
@@ -233,13 +233,6 @@ jobs:
           token: ${{ secrets.NPM_ACCESS_TOKEN }}
           tag: latest
           package: packages/common/package.json
-      - name: "publish to npm: common-public"
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-        uses: JS-DevTools/npm-publish@v1
-        with:
-          token: ${{ secrets.NPM_ACCESS_TOKEN }}
-          tag: latest
-          package: packages/common-public/package.json
```
Comment (wentokay) [inline .github/workflows/pull_requests_and_merges.yml:242]: do we still need to publish `@coral-xyz/common-public` @armaniferrante?

Comment (armaniferrante) [reply .github/workflows/pull_requests_and_merges.yml:242]: nope

Comment (wentokay) [reply .github/workflows/pull_requests_and_merges.yml:242]: it's published as a react-xnft dependency

https://unpkg.com/react-xnft@0.2.0-latest.543/package.json

although it's using the asterisk which probably shouldn't really be in package.json on npm

looking into it

**Comment by wentokay** [comment]
closing as this is going to quickly far behind master

I'll come back with a complete solution that could be merged immediately once everything has been prepared

**Pull request closed without merging**